src/h5web/vis-packs/core/shared/Html.tsx

```diff
@@ -70,20 +85,32 @@ const Html = forwardRef<HTMLDivElement, HtmlProps>((props, ref) => {
       if (parentElement) {
         el.remove();
       }
+
       ReactDOM.unmountComponentAtNode(el);
     };
-  }, [el, parentElement]); // eslint-disable-line react-hooks/exhaustive-deps
+  }, [el, parentElement]);
 
+  // Update size and overflow of container `div`
   useLayoutEffect(() => {
+    el.style.width = `${width}px`;
+    el.style.height = `${height}px`;
+    el.style.overflow = followCamera ? 'hidden' : 'visible';
+  }, [el, width, height, followCamera]);
+
+  // Render inner `div`
+  useEffect(() => {
+    group.current.updateMatrixWorld(); // make sure we get the group's most up to date position
+
     ReactDOM.render(
       <div
-        ref={ref}
+        ref={innerRef}
         className={className}
         style={{
           position: 'absolute',
           top: 0,
           left: 0,
-          pointerEvents: 'none', // let pointer events pass through to canvas
+          transformOrigin: 'left top',
+          transform: getInnerDivTransform(),
```

Comment:
As mentioned, we now set the `transform` on the inner `div`. We compute it here right away to avoid a positioning glitch.